web/src/ShareSnapshotModal.scss

```diff
@@ -0,0 +1,60 @@
+@import "constants";
+
+.modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width:100%;
+  height: 100%;
+  background: rgba(0, 0, 0, 0.6);
+}
+
+.modal-main {
+  position:fixed;
+  background: $color-white;
+  color: $color-gray-darkest;
+  width: 80%;
+  height: auto;
+  top:50%;
+  left:50%;
+  transform: translate(-50%,-50%);
+}
+
+.display-block {
```

Comment:
i would usually call this u-displayBlock / u-displayNone

in BEM convention, "display-none" typically means "in the display component, there's a sub-element called block", which is obviously not what you mean here